tools/linter/codespell/.codespell.ignorewords

```diff
@@ -0,0 +1,2 @@
+keypair
+keypairs
```

Comment:
What is this based on?

Reply:
Reverted, a codespell tool widely used for linting codes.